pages/common/azure-func.md

```diff
@@ -0,0 +1,29 @@
+# func
+
+> Azure Functions Core Tools lets you develop and test your functions on your local computer from the command prompt or terminal.
+> Your local functions can connect to live Azure services, and you can deploy a function app to your Azure subscription.
+> More information: <https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local>.
+
+- Create a new functions project:
+
+`func init {{project}}`
+
+- Create a new function:
+
+`func new`
+
+- Run functions locally:
+
+`func start`
+
+- Publish your local code to a function app in Azure:
```

Comment:
We don't use `app` as acronym.
Is not called `Azure Function`? like this:
```suggestion
- Publish your local code as an Azure Function:
```

Reply:
@navarroaxel: It depends on the language that's used in the ecosystem from which the command originates. In this case, a [function app appears to be an official thing](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-function-app-portal).

Also note that generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:

What about this?
```suggestion
- Publish local code to a function app in Azure:
```